### PR TITLE
feature/add-language-to-pages

### PR DESCRIPTION
**Pull request by mickietrill**

### CODE REVIEW

File: src/components/languageMenu/LanguageMenuPure.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import OutsideClickHandler from 'react-outside-click-handler'
+import LanguageContext from 'src/contexts/language'
+import './language-menu.less'
+import configPages from 'src/config'
+
+const LanguageMenuPure = ({
+  closeLanguageMenu,
+  toggleLanguageMenu,
+  isLanguageMenuShown
+}) => {
```
Comment (Evoltic) [inline src/components/languageMenu/LanguageMenuPure.js]: Could you please make a destructive inside the function if it's doesn't fit a single row. 
(props) => {  
  const { one, two, three } = props 
}

File: src/components/languageMenu/LanguageMenuPure.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  const locationPathname = window.location.href
+    .split('/')
+    .filter(function(element) {
+      return element !== ''
+    })
+    .pop()
```
Comment (Evoltic) [inline src/components/languageMenu/LanguageMenuPure.js]: Could you move this logic to a function and name it? "getCurrentPageName" for example

File: src/components/languageMenu/LanguageMenuPure.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  if (
+    configPages[locationPathname] &&
+    configPages[locationPathname].languages !== -1
```
Comment (Evoltic) [inline src/components/languageMenu/LanguageMenuPure.js]: Could you move long lines outside of if statement and name it accordingly what it does

File: src/components/languageMenu/LanguageMenuPure.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  ) {
+    const currentLanguage = LanguageContext._currentValue
+    const currentAvailableLanguage = configPages[locationPathname].languages
+    const domainHref = 'https://platinum.fund/'
+
+    const currentLanguageContent = (
```
Comment (Evoltic) [inline src/components/languageMenu/LanguageMenuPure.js]: Make a Component please and move it outside of the function. And better to another file

File: src/components/languageMenu/LanguageMenuPure.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+          alt="Language en"
+          className="languages-menu__option-image"
+          src={require('./images/languages/' + currentLanguage + '.png')}
+          data-option="en"
+        />
+        <p className="languages-menu__option-name">{currentLanguage}</p>
+      </button>
+    )
+
+    const currentLanguageList = currentAvailableLanguage.map(
```
Comment (Evoltic) [inline src/components/languageMenu/LanguageMenuPure.js]: Make a Component please and move it outside of the function. And better to another file

File: src/components/languageMenu/language-menu.less [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+  .languages-menu__options {
+    position: absolute;
+    top: 3em;
+    right: 0;
+    display: none;
+    border-radius: 3px;
+    background-color: #fff;
+    box-shadow: 0 3px 6px rgba(0, 0, 0, 0.1), 0 3px 6px rgba(0, 0, 0, 0.13);
+    &_hidden {
```
Comment (Evoltic) [inline src/components/languageMenu/language-menu.less]: If I'm not wrong, according to BEM it should be "&--hidden"

File: src/components/languageMenu/language-menu.less [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    }
+  }
+
+  .languages-menu__current-option {
+    all: unset;
+    .languages-menu__option();
+    color: inherit;
+    cursor: pointer;
+    position: relative;
+    z-index: 1;
```
Comment (Evoltic) [inline src/components/languageMenu/language-menu.less:44]: position: relative;
z-index: 1;  
color: inherit;
cursor: pointer;

File: src/components/navbar/NavbarPure.js [hunk 8]
```diff
@@ -1,81 +1,103 @@
...
+import LanguageMenu from '../languageMenu'
 
 const Navbar = ({ closeMenu, toggleMenu, isMenuShown }) => {
+  const currentLanguage = LanguageContext._currentValue
+  const domainHref = 'https://platinum.fund/'
+  const hrefWithLanguage = pageName => {
+    if (
+      configPages[pageName] &&
+      configPages[pageName].languages &&
+      configPages[pageName].languages.indexOf(currentLanguage) !== -1
```
Comment (Evoltic) [inline src/components/navbar/NavbarPure.js:19]: Move it to statement please

**Review by Evoltic — CHANGES_REQUESTED** [review]

File: src/components/languageMenu/LanguageMenuPure.js (hunk 1 above)
Comment (mickietrill) [reply src/components/languageMenu/LanguageMenuPure.js]: done

File: src/components/languageMenu/LanguageMenuPure.js (hunk 2 above)
Comment (mickietrill) [reply src/components/languageMenu/LanguageMenuPure.js]: done

File: src/components/languageMenu/LanguageMenuPure.js (hunk 3 above)
Comment (mickietrill) [reply src/components/languageMenu/LanguageMenuPure.js]: done

File: src/components/languageMenu/LanguageMenuPure.js (hunk 4 above)
Comment (mickietrill) [reply src/components/languageMenu/LanguageMenuPure.js]: done

File: src/components/languageMenu/LanguageMenuPure.js (hunk 5 above)
Comment (mickietrill) [reply src/components/languageMenu/LanguageMenuPure.js]: done

File: src/components/languageMenu/language-menu.less (hunk 6 above)
Comment (mickietrill) [reply src/components/languageMenu/language-menu.less]: done

File: src/components/languageMenu/language-menu.less (hunk 7 above)
Comment (mickietrill) [reply src/components/languageMenu/language-menu.less:44]: done

File: src/components/languageMenu/language-menu.less (hunk 6 above)
Comment (mickietrill) [reply src/components/languageMenu/language-menu.less]: "-" -  A hyphen is used to separate words in names
"_" - The value of the modifier is separated from the name of the modifier by one underscore

File: src/components/navbar/NavbarPure.js (hunk 8 above)
Comment (mickietrill) [reply src/components/navbar/NavbarPure.js:19]: it makes no sense, it will add more lines of code that is not so logical

**Review by mickietrill — COMMENTED** [review]
The proposed changes are made.

File: src/components/navbar/NavbarPure.js (hunk 8 above)
Comment (Evoltic) [reply src/components/navbar/NavbarPure.js:19]: It's better to understand thats going on

const isLanguageExist = configPages[pageName] && 
  configPages[pageName].languages && 
  configPages[pageName].languages.indexOf(currentLanguage) !== -1

if (isLanguageExist ) { ... }

**Pull request merged**